2021-04-28 18:12:14,743 INFO    MainThread:907134 [wandb_setup.py:_flush():69] setting env: {}
2021-04-28 18:12:14,744 INFO    MainThread:907134 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-28 18:12:14,744 INFO    MainThread:907134 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-28 18:12:14,744 INFO    MainThread:907134 [wandb_init.py:_log_setup():336] Logging user logs to /home/miguel/Desktop/NLP/MUStARD/wandb/run-20210428_181214-q4o7khbt/logs/debug.log
2021-04-28 18:12:14,744 INFO    MainThread:907134 [wandb_init.py:_log_setup():337] Logging internal logs to /home/miguel/Desktop/NLP/MUStARD/wandb/run-20210428_181214-q4o7khbt/logs/debug-internal.log
2021-04-28 18:12:14,745 INFO    MainThread:907134 [wandb_init.py:init():369] calling init triggers
2021-04-28 18:12:14,745 INFO    MainThread:907134 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-28 18:12:14,745 INFO    MainThread:907134 [wandb_init.py:init():418] starting backend
2021-04-28 18:12:14,746 INFO    MainThread:907134 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-28 18:12:14,765 INFO    MainThread:907134 [backend.py:ensure_launched():123] starting backend process...
2021-04-28 18:12:14,809 INFO    MainThread:907134 [backend.py:ensure_launched():128] started backend process with pid: 1685270
2021-04-28 18:12:14,814 INFO    MainThread:907134 [wandb_init.py:init():423] backend started and connected
2021-04-28 18:12:14,840 INFO    MainThread:907134 [wandb_init.py:init():465] updated telemetry
2021-04-28 18:12:14,860 INFO    MainThread:907134 [wandb_init.py:init():484] communicating current version
2021-04-28 18:12:15,863 INFO    MainThread:907134 [wandb_init.py:init():489] got version response 
2021-04-28 18:12:15,864 INFO    MainThread:907134 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-28 18:12:16,264 INFO    MainThread:907134 [wandb_init.py:init():522] starting run threads in backend
2021-04-28 18:12:21,321 INFO    MainThread:907134 [wandb_run.py:_console_start():1524] atexit reg
2021-04-28 18:12:21,323 INFO    MainThread:907134 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-28 18:12:21,342 INFO    MainThread:907134 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-28 18:12:21,387 INFO    MainThread:907134 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-28 18:12:21,387 INFO    MainThread:907134 [wandb_init.py:init():546] run started, returning control to user process
2021-04-28 18:12:21,388 INFO    MainThread:907134 [wandb_run.py:_config_callback():749] config_cb None None {'config_key': 'tav'}
2021-04-28 18:13:04,266 INFO    MainThread:907134 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 255
2021-04-28 18:13:04,293 INFO    MainThread:907134 [wandb_run.py:_restore():1466] restore
2021-04-28 18:13:06,768 INFO    MainThread:907134 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 22432
  total_bytes: 22432
}

2021-04-28 18:13:07,369 INFO    MainThread:907134 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 7
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 22432
  total_bytes: 30606
}

2021-04-28 18:13:07,763 INFO    MainThread:907134 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 7
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 22432
  total_bytes: 30606
}

2021-04-28 18:13:08,242 INFO    MainThread:907134 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 7
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 22432
  total_bytes: 30606
}

2021-04-28 18:13:08,517 INFO    MainThread:907134 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 7
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 30606
  total_bytes: 30606
}

2021-04-28 18:13:08,816 INFO    MainThread:907134 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 7
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 30606
  total_bytes: 30606
}

2021-04-28 18:13:08,949 INFO    MainThread:907134 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 7
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 30606
  total_bytes: 30606
}

2021-04-28 18:13:09,068 INFO    MainThread:907134 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 7
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 30606
  total_bytes: 30606
}

2021-04-28 18:13:10,424 INFO    MainThread:907134 [wandb_run.py:_show_files():1838] logging synced files
